src/libraries/System.Net.Mail/tests/Unit/System.Net.Mail.Unit.Tests.csproj

```diff
@@ -159,7 +159,7 @@
              Link="Common\System\HexConverter.cs" />
   </ItemGroup>
   <!-- Unix specific files -->
-  <ItemGroup Condition="'$(TargetsUnix)'=='true'">
+  <ItemGroup Condition="'$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true'">
```

Comment:
All of the source files in System.Net.Mail.Unit.Tests.csproj and System.Net.Mail.Functional.Tests.csproj (apart from AssemblyInfo.cs) could be excluded now that the library throws PNSE on Browser.